pkg/controller/options.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2020 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+import (
+	"time"
+)
+
+// Options TODO
+type Options struct {
```

Comment:
this might be useful to pass crossplane specific options while initializing the controller manager. An example can be `crossplane/cmd/core/core.go`:

```go
	mgr, err := controller.NewManager(cfg, controller.Options{
		ManagerName:             "core",
		ManagerNamespace:        c.Namespace,
		LeaderElection:          true,
		LeaderElectionNamespace: c.Namespace,
		SyncPeriod:              &c.Sync,
	})
```